packages/web/src/components/modal-drawer/ModalDrawer.tsx

```diff
@@ -7,15 +15,21 @@ import { useIsMobile } from 'hooks/useIsMobile'
 import styles from './ModalDrawer.module.css'
 
 type ModalDrawerProps = ModalProps &
-  DrawerProps & { useGradientTitle?: boolean }
+  DrawerProps & {
+    useGradientTitle?: boolean
```

Comment:
can we rename this? `isGradientTitle` or just `gradientTitle` or something

Reply:
yeah def, actually id like to drop this entirely, lol